components/Button.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import {
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  Image,
+  ImageSourcePropType,
+  TextStyle,
+  StyleProp,
+  ViewStyle,
+  GestureResponderEvent,
+} from 'react-native';
+
+const styles = StyleSheet.create({
+  buttonText: {
+    color: '#2c2c2c',
+    textAlign: 'left',
+    fontSize: 16,
+    fontWeight: 'bold',
+    textAlignVertical: 'center',
+  },
+  button: {
+    height: 50,
+    backgroundColor: 'white',
+    padding: 10,
+    flexDirection: 'row',
+  },
+  icon: {
+    width: 28,
+    height: 28,
+    marginLeft: 10,
+    marginRight: 30,
+  },
+});
+
+type Props = {
```

Comment:
Also, I think it would be a good idea to export the types from this component since it would be a lot easier to test use their props out of the library instead of rewriting these props again.

What do you think?

Reply:
definitely